.travis.yml

```diff
@@ -19,16 +28,14 @@ cache:
   - "$HOME/.gradle/caches/modules-2"
   - "$HOME/.gradle/caches/package-lists"
   - "$HOME/.gradle/caches/sphinx-binary"
+  - "$HOME/.jdk"
 env:
   global:
   - "_JAVA_OPTIONS=-Djavax.net.ssl.trustStore=/etc/ssl/certs/java/cacerts"
-before_install:
-- "./gradlew --version"
 install:
 - true
 script:
+- "./gradlew --version"
 - "./gradlew --no-daemon --stacktrace --warning-mode=all -Pcoverage checkstyle check build"
 before_cache:
 - find $HOME/.gradle/caches -name '*.lock' -delete
-after_success:
-- bash <(curl -s https://codecov.io/bash)
```

Comment:
Are you going to re-enable this later, or only for a certain Java version?

Reply:
I figured we should upload coverage once per build. Do you think it's good to upload for all of them?

Reply:
I guess we could just pick and upload only one? (Could disable coverage for others for faster builds)

Reply:
Yeah - I do that above in the matrix :) But good idea to disable coverage on the other builds